### PR TITLE
Create package.json, add config to be a global module, save dependencies

### PR DESCRIPTION
**Pull request by eafelix**
We add the package.json file and the `bin` configuration to have it installed globally for use it directly wherever you want in your system

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,11 +5,12 @@ node.js command line bitcoin HD wallet, BIP 44 compatible.
 
 ## Requirements
 
-### NPM modules
+Until we have posted the module in the NPM registry, for installing it globally using the command `npm install -g waljs`,
+we must clone the repository and execute `npm install` and then `npm link`, then will be able to use it directly in the cmd.
```
Comment (gabmontes) [inline README.md]: Rephrase to speak to the user/reader of the repo and not to ourselves: just give instructions on how to install and run.

File: README.md [hunk 2]
```diff
@@ -5,11 +5,12 @@ node.js command line bitcoin HD wallet, BIP 44 compatible.
 
 ## Requirements
 
-### NPM modules
+Until we have posted the module in the NPM registry, for installing it globally using the command `npm install -g waljs`,
```
Comment (gabmontes) [inline README.md]: Remove this phrase. We could change the README again once we publish waljs as a package.

File: README.md [hunk 3]
```diff
@@ -5,11 +5,12 @@ node.js command line bitcoin HD wallet, BIP 44 compatible.
...
+we must clone the repository and execute `npm install` and then `npm link`, then will be able to use it directly in the cmd.
 
-Install several npm modules for node.js:
-
-	$ npm install async commander bitcore-lib bitcore-mnemonic bitcoind-rpc bitcore-p2p
+```BASH
+$ waljs
```
Comment (gabmontes) [inline README.md]: Better to explicitly say the user has to `sudo npm link` and then `waljs`.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "waljs",
+  "version": "0.0.1",
+  "description": "node.js command line bitcoin HD wallet, BIP 44 compatible.",
+  "main": "wal.js",
+  "bin": {
+    "waljs": "./wal.js"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (gabmontes) [inline package.json]: Remove this line.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bloq/waljs.git"
+  },
+  "keywords": [
+    "wallet",
+    "blockchain",
+    "bloq"
+  ],
+  "author": "Jeff Garzik",
```
Comment (gabmontes) [inline package.json]: Add Jeff's email: `Jeff Garzik <jeff@bloq.com>`.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  "license": "ISC",
```
Comment (gabmontes) [inline package.json]: Remove and check with Jeff what licensing is most appropriate.

**Review by gabmontes — CHANGES_REQUESTED** [review]
With some cleanup will be an awesome PR!

**Comment by jgarzik** [comment]
Agree w/ @gabmontes 's comments.  Thanks for the contribution & thanks for the review.

File: package.json (hunk 6 above)
Comment (gabmontes) [reply package.json]: @jgarzik what license type should be used for waljs?

Comment (jgarzik) [reply package.json]: @gabmontes principle of least surprise:  same as bitcore-lib (MIT)

**Review by gabmontes — APPROVED** [review]

**Pull request merged**